opencti-platform/opencti-graphql/src/modules/auth/auth.graphql

```diff
@@ -0,0 +1,30 @@
+input AskSendOtpInput {
+  email: String!
+}
+
+input VerifyOtpInput {
+  email: String!
+  otp : String!
+}
+
+input Verify2faInput {
+  code: String!
+  email: String!
+}
+
+input ChangePasswordInput {
+  email: String!
+  newPassword: String!
+}
+
+type VerifyOtp {
+  otp_activated: Boolean!
+}
+
+## Mutations
+type Mutation {
+  askSendOtp(input: AskSendOtpInput!): Boolean @public
+  verifyOtp(input: VerifyOtpInput!): VerifyOtp @public
+  verify2fa(input: Verify2faInput!): Boolean @public
+  changePassword(input: ChangePasswordInput!): Boolean @public
```

Comment:
I don't think this method should be public, how do we prevent anyone to change password of other users ?